geode-core/src/main/java/org/apache/geode/internal/cache/EvictableRegion.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.cache;
+
+import org.apache.geode.StatisticsFactory;
+import org.apache.geode.cache.EvictionAttributes;
+import org.apache.geode.internal.cache.eviction.EvictionController;
+
+public interface EvictableRegion {
+  EvictionAttributes getEvictionAttributes();
+
+  boolean getOffHeap();
```

Comment:
this was a preexisting method on the implementation classes that I just added to this new interface. So for this PR I don't plan on renaming it. I think it comes from the external RegionAttributes interface so renaming it could break existing user's code.

Reply:
makes sense